### PR TITLE
feat: add setImage api

### PR DESCRIPTION
**Pull request by ufkaya**
References [CAR-6437](https://autoricardo.atlassian.net/browse/CAR-6437)

### CODE REVIEW

**Review by GutierrezGo — APPROVED** [review]

File: src/services/dealer.ts [hunk 1]
```diff
@@ -225,3 +225,31 @@ export const setLogo = async ({
...
   }
 }
+
+export const setImage = async ({
+  dealerId,
+  image,
+  options = {},
+}: {
+  dealerId: number
+  image: string
```
Comment (Averethel) [inline src/services/dealer.ts:235]: Can we call it `imageUrl` or `imageS3Key`? When I read image here I expect an image object.

Comment (masone) [reply src/services/dealer.ts]: keep it in sync with the other places where we upload images :)

Comment (ufkaya) [reply src/services/dealer.ts]: what do you mean? Lets keep `image` since this is the field name in BE

Comment (masone) [reply src/services/dealer.ts:235]: yeah, I guess that's what I mean... I'd value consistency with other api calls and backends naming over clarity

**Pull request merged**